macro_utils_h_generator/macro_utils.tt

```diff
@@ -305,7 +305,7 @@ MACRO(0, __VA_ARGS__)
 <#for(var i=1;i<=nArithmetic;i++){#>
 #define MU_DO_<#=i#>(MACRO, ...) \
 MACRO(<#=i#>, __VA_ARGS__) \
-DO_<#=i-1#>(MACRO, __VA_ARGS__)
+MU_DO_<#=i-1#>(MACRO, __VA_ARGS__)
```

Comment:
I would suggest adding a test for this. There is a test project that could just make use once of MU_DO to make sure it works.